kruize-clowdapp.yaml

```diff
@@ -1,193 +1,287 @@
+            image: ${KRUIZE_IMAGE}:${KRUIZE_IMAGE_TAG}
+            command: ["sh"]
+            args:
+              [
+                "-c",
+                "export DB_CONFIG_FILE=${ACG_CONFIG} && bash target/bin/Autotune",
+              ]
+            machinePool: ${MACHINE_POOL_OPTION}
+            resources:
+              requests:
+                cpu: ${KRUIZE_CPU_REQUEST}
+                memory: ${KRUIZE_MEMORY_REQUEST}
+              limits:
+                cpu: ${KRUIZE_CPU_LIMIT}
+                memory: ${KRUIZE_MEMORY_LIMIT}
+            env:
+              - name: CLOWDER_ENABLED
+                value: ${CLOWDER_ENABLED}
+              - name: AUTOTUNE_SERVER_PORT
+                value: ${KRUIZE_PORT}
+              - name: AUTH_TOKEN
+                value: ""
+              - name: LOGGING_LEVEL
+                value: ${KRUIZE_LOGGING_LEVEL}
+              - name: ROOT_LOGGING_LEVEL
+                value: "error"
+              - name: dbdriver
+                value: "jdbc:postgresql://"
+              - name: clustertype
+                value: "kubernetes"
+              - name: k8stype
+                value: "openshift"
+              - name: authtype
+                value: "openshift"
+              - name: monitoringagent
+                value: "prometheus"
+              - name: monitoringservice
+                value: "prometheus-k8s"
+              - name: monitoringendpoint
+                value: "prometheus-k8s"
+              - name: savetodb
+                value: "true"
+              - name: hibernate_dialect
+                value: "org.hibernate.dialect.PostgreSQLDialect"
+              - name: hibernate_driver
+                value: "org.postgresql.Driver"
+              - name: hibernate_c3p0minsize
+                value: "5"
+              - name: hibernate_c3p0maxsize
+                value: "10"
+              - name: hibernate_c3p0timeout
+                value: "300"
+              - name: hibernate_c3p0maxstatements
+                value: "100"
+              - name: hibernate_hbm2ddlauto
+                value: "none"
+              - name: hibernate_showsql
+                value: "false"
+              - name: hibernate_timezone
+                value: "UTC"
+              - name: SSL_CERT_DIR
+                value: ${SSL_CERT_DIR}
+      services:
+        - name: kruize-ui-nginx-service
+          type: NodePort
+          ports:
+            - name: http
+              port: 8080
+              targetPort: 8080
+
+      pods:
+        - name: kruize-ui-nginx-pod
+          containers:
+            - name: kruize-ui-nginx-container
+              image: quay.io/kruize/kruize-ui:0.0.3
+              imagePullPolicy: Always
+              env:
+                - name: KRUIZE_UI_ENV
+                  value: "production"
+          volumeMounts:
+            - name: nginx-config-volume
+              mountPath: /etc/nginx/nginx.conf
+              subPath: nginx.conf
```

Comment:
You cannot directly declare pods here, You will need to create another deployment. Please refer CloudApp API reference doc - https://consoledot.pages.redhat.com/clowder/dev/api_reference.html#k8s-api-github-com-redhatinsights-clowder-apis-cloud-redhat-com-v1alpha1-podspec